README.md

```diff
@@ -194,17 +192,73 @@ crafts = [
 _The first craft requires a stone axe to craft the item referenced "1", the second craft enable us to cook a cooked_porkchop into a coal, 
 and the third one is used to craft the item referenced "2" with three sticks aligned in a vertical centered line (notice the whitespaces before and after the "a")_
 
+### Affix
+
+Affix are modifiers who can be applied to item. They are pre-registered in pools (not items pools :wink:).
+The root is `affixes`, it contains a list of affix groups. a group is categorized by is `group` name : it's a schematic name only used for configuration like
```

Comment:
- Precise avant cette phrase que les affix se configurent dans un JSON ou autre ! Car la tu parles de root mais on sait pas ce que ca veut dire  
- Majuscule a `a group` :)  
- `by its group name` et non is. is c'est le verbe `to be`, tandis que `its` c'est 'son' (un peu comme si tu mettais `it's group`, comme `Hugo's group`)  
- Pour plus de clarte je conseille de separer la derniere partie: `...only used for configuration. (group name examples: damage,...`